sway-lsp/src/test_utils.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) fn e2e_language_dir() -> PathBuf {
 pub(crate) fn e2e_test_dir() -> PathBuf {
     sway_workspace_dir()
         .join(e2e_language_dir())
-        .join("struct_field_access")
+        .join("prelude_access2")
```

Comment:
Was this failing on the old e2e test dir, or is this just ensuring it's testing with preludes?

Reply:
it was just for testing preludes locally. I'll change it back so it's less noise for the PR